src/Stripe.Tests.XUnit/Stripe.Tests.XUnit.csproj

```diff
@@ -19,6 +19,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
+    <PackageReference Include="Stylecop.Analyzers" Version="1.0.2">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
```

Comment:
These two lines instruct NuGet to not add `Stylecop.Analyzers` as a package dependency, so that it's not propagated to Stripe.net users. Cf. https://github.com/NuGet/Home/issues/7051.